src/App/_tests_/Login.test.js

```diff
@@ -12,6 +12,9 @@ const props = {
 const shallowed = shallow(<Login {...props} />);
 const instance = shallowed.instance();
 
+// eslint-disable-next-line react/display-name
+jest.mock('../Components/Common/NavProfile/navbar', () => () => (<div id="mock-navbar">Mock navbar</div>));
```

Comment:
@nkpremices why disable this rule?

Reply:
@MemunaHaruna `eslint` was complaining about this rule. I tried to fix it but I couldn't. I even went through the documentation bu still couldn't get how to solve it.